inventory/sample/group_vars/k8s-cluster.yml

```diff
@@ -192,6 +192,9 @@ ingress_nginx_enabled: false
 # ingress_nginx_namespace: "ingress-nginx"
 # ingress_nginx_insecure_port: 80
 # ingress_nginx_secure_port: 443
+# ingress_nginx_configmap: {}
```

Comment:
can you add some sample how dict should look like, so people don't ask questions how to write etc...

Reply:
Copy ConfigMap example from:
* https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/configmap.md
* https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/exposing-tcp-udp-services.md